### DISCUSSION TITLE
Is there a WakaTime importer?

### DISCUSSION THREAD

**Discussion by itzTheMeow** [ActivityWatch/activitywatch#1049] [General]
It seems you can export all of your wakatime data as heartbeats, so it should be fairly simple to convert that to aw events. I'm curious if there is already an importer for this out there. (i couldn't find one with googling around)
If not, I'd like to make an attempt at creating one.

**Comment by 0xbrayo** [comment]
I don't believe there is, feel free to get started on one!

**Comment by 0xbrayo** [comment]
I made one here [repo]( https://github.com/brayo-pip/aw-import-wakatime). PRs open
